.github/workflows/build-and-deploy.yml

```diff
@@ -41,15 +38,15 @@ jobs:
           git config --global user.name "David Négrier"
           git checkout master
           git add .
-          git commit -am "Adding files"
+          git commit -am "Generate map for Github Pages"
 
       - name: Deploy
         uses: JamesIves/github-pages-deploy-action@releases/v3
         if: github.ref == 'refs/heads/master'
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           BRANCH: gh-pages # The branch the action should deploy to.
-          FOLDER: . # The folder the action should deploy.
+          FOLDER: public/ # The folder the action should deploy.
```

Comment:
Could we try to remove the "mv", keep things in "dist", and do not "git commit" (I can't remember why we needed to do that)